Transformers/Transformers.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<p>If these are all "distilbert", how are we supposed to use them separately? The naming convention kungfu is invisible to the person reading this notebook. We should either pass the filenames from here, or have some other way for that to be made visible to the user e.g. via a return or part of the <code>hinglishbert</code> object</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Currently the name for the model is being given bert_timestamp or distilbert_timestamp which is being logged in the logfile. </p><p>Are you saying that we should also give the option to override out default names with their name in an additional parameter? </p><p> </p>